stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ui/MessagesActivity.kt

```diff
@@ -92,7 +94,10 @@ class MessagesActivity : AppCompatActivity() {
         val channelId = intent.getStringExtra(KEY_CHANNEL_ID) ?: return
 
         setContent {
-            ChatTheme(dateFormatter = ChatApp.dateFormatter) {
+            ChatTheme(
+                dateFormatter = ChatApp.dateFormatter,
+                deletedMessagesVisibility = DeletedMessagesVisibility.OWN
```

Comment:
This feels like 2 sources of truth. If the `deletedMessagesVisibility` param in `ChatTheme` has a different value here, then nothing will change as we have hardcoded value above passed to `MessagesViewModelFactory`. What about removing the param from `ChatTheme` completely?

Reply:
It's provided by the theme as our users may need to use it somewhere in their custom logic. Also because it passes it down to the MessagesScreen and hence the ViewModel.
In case our users use stateless components, they'll need a central place to fetch the value from. Which the ChatTheme can be

Reply:
Removed from theme, applied only in the ViewModel.